changelog/security/2023-09-08-nvidia-drivers-535.104.05.md

```diff
@@ -0,0 +1 @@
+- grub ([CVE-2023-25515](https://nvd.nist.gov/vuln/detail/CVE-2023-25515), [CVE-2023-25516](https://nvd.nist.gov/vuln/detail/CVE-2023-25515))
```

Comment:
`s/grub/nvidia-drivers/` ?

Reply:
The second CVE should be like `[CVE-2023-25516](https://nvd.nist.gov/vuln/detail/CVE-2023-25516)`.